pkg/apis/extensions/v1alpha1/types_worker.go

```diff
@@ -198,6 +198,9 @@ type ClusterAutoscalerOptions struct {
 type NodeTemplate struct {
```

Comment:
As discussed offline, the doc string usage of `NodeTemplate` should be amended [here](https://github.com/gardener/gardener/blob/68d29023bcc82a3732546804fa7931f70a87d74f/pkg/apis/extensions/v1alpha1/types_worker.go#L161).